wordpress-mobile/WordPress-iOS
# Use FormatterKit to display short dates.

Fixes #5217 

To test: 

 - Go to the reader and scroll around and check the date formats being displayed.
 - Go to post list and check the dates on the cells.
 - Go to pages list and check the dates on the headers

If possible switch to another language that you know and it should be supported for the app and see if the translation shows up

Needs review: @aerych
